integration/helper_test.go

```diff
@@ -160,12 +165,14 @@ func createTestPolicy(ctx context.Context, c *envconf.Config, namespace string,
 ) (*v1alpha1.SecurityPolicy, error) {
 	var policy v1alpha1.SecurityPolicy
 
-	policy.Name = testSecurityPolicy
+	policy.Name = testSecurityPolicy + fmt.Sprintf("%d", rand2.Int())
 	policy.Namespace = namespace
 	policy.Spec = v1alpha1.SecurityPolicySpec{
-		Name:   testSecurityPolicy,
-		Active: true,
-		State:  "active",
+		SecurityPolicySpec: model.SecurityPolicySpec{
+			Name:   testSecurityPolicy + testSecurityPolicy + uuid.New().String()[:8],
```

Comment:
Why are you using just first 8 characters of string here?

Reply:
no specific reason. i just used 8 characters to prevent possible naming overflows etc.

Reply:
If we use just 8 characters, not sure it will be always a unique string.

Reply:
updated as you suggested 👍 thanks!